examples/support-vector-machine/script.jl

```diff
@@ -49,7 +48,7 @@ model = svmtrain(kernelmatrix(k, x_train), y_train; kernel=LIBSVM.Kernel.Precomp
 y_pred, _ = svmpredict(model, kernelmatrix(k, x_train, x_test));
 
 # Visualize prediction on a grid:
-plot(; lim=extrema(test_range), aspect_ratio=1)
+plot(; xlim=extrema(test_range), ylim=extrema(test_range), aspect_ratio=1)
```

Comment:
I guess you want
```suggestion
plot(; widen=false, aspect_ratio=1)
```
?

Reply:
No, that doesn't fix it.

Reply:
Ah, I guess you might have to specify the keyword argument when you actually plot data since otherwise subsequent calls might update the limits. I missed that there's nothing plotted in this call here.

Reply:
No, widen isn't actually needed. Looks like updating Plots.jl seems to have fixed it though :sweat_smile: